components/service/firefox-accounts/src/test/java/mozilla/components/service/fxa/FxaDeviceConstellationTest.kt

```diff
@@ -232,6 +239,68 @@ class FxaDeviceConstellationTest {
         assertEquals(ConstellationState(currentDeviceExpired.into(), listOf(testDevice2.into())), constellation.state())
     }
 
+    @Test
+    @ExperimentalCoroutinesApi
+    fun `polling for commands with real account`() = runBlockingTest {
```

Comment:
Any specific reason you're using `runBlockingTest` instead of `runBlocking`? It's mainly useful if you need to skip over any delays in your test code, which I don't think is the case here.

Reply:
See all the other tests in this file for an established pattern for dealing with coroutines.